drivers/counter/counter_cc23x0_rtc.c

```diff
@@ -153,6 +154,23 @@ static uint32_t counter_cc23x0_get_pending_int(const struct device *dev)
 	return -ESRCH;
 }
 
+#ifdef CONFIG_PM_DEVICE
+
+static int rtc_cc23x0_pm_action(const struct device *dev, enum pm_device_action action)
+{
+	switch (action) {
+	case PM_DEVICE_ACTION_SUSPEND:
+		return 0;
+	case PM_DEVICE_ACTION_RESUME:
+		counter_cc23x0_get_pending_int(dev);
+		return 0;
+	default:
+		return -ENOTSUP;
+	}
+}
```

Comment:
I could use a little more detail on how this is adding PM support?

Reply:
RTC is part of AON domain , so it kept active during sleep and is used for wakeup from sleep. Once recover from sleep is pulling missed interrupt while reconfigured to normal.